week1/homework.md

```diff
@@ -48,61 +48,57 @@ Now add a `.gitignore` file with these two lines:
 
 `.gitignore` will make sure we don't commit all the files in the `node_modules` folder, which can get huge.
 
-We are going to be using `nodemon` to run our application. `nodemon` will watch for file changes, and when a file is changed it will rerun our app. To install `nodemon` run
+The first step is to install [express](https://expressjs.com/) which is a javascript web framework.
 
-`npm install --save-dev nodemon`
+Install `express` with the following command
 
-The `--save-dev` will save the `nodemon` as a developer dependency
+    npm install -s express
 
-Also install `express` with the following command
+Adding the `-s` will save `express` as a dependency.
 
-`npm install -s express`
+To make our developing lives easier, we will use `nodemon` to run our web server (express). `nodemon` will watch for file changes, and when a file is changed it will rerun our app. To install `nodemon` run
 
-`-s` will save `express` as a dependency
+    npm install --save-dev nodemon
+
+The `--save-dev` will save the `nodemon` as a developer dependency.
+
+And now to ensure all your packages are installed for good measure, run:
+
+    npm install
 
 ### package.json
 
 Inside the `package.json` file we can define scripts that can be run. There should already be a `test` script. In the same fashion, add two more scripts:
 
-1. `start` will run the command `node src/backend/index.js`. This we use when we want to start up the server.
-2. `dev` will run the command `nodemon src/backend/index.js`. This we use when we want to develop our application.
+1. `start` will run the command `node src/backend/server.js`. This we use when we want to start up the server.
+2. `dev` will run the command `nodemon src/backend/server.js`. This we use when we want to develop our application.
```

Comment:
There is no start or dev script. We should add those.

Reply:
Do we want the students to add these themselves, or should we add the scripts for them?

Reply:
Omg first day back at the office 🤦‍♂️They add it them selves. Ignore this